### PR TITLE
RHDEVDOCS-3567 - Release notes for v1.3.2

### PR DESCRIPTION
**Pull request by DebarghoGhosh**
Aligned team: Dev Tools
OCP version for cherry-picking: enterprise-4.9
JIRA issues:https://issues.redhat.com/browse/RHDEVDOCS-3567

Preview pages: https://deploy-preview-40110--osdocs.netlify.app/openshift-enterprise/latest/cicd/gitops/gitops-release-notes

SME+QE review: 
Peer-review:

### CODE REVIEW

**Comment by DebarghoGhosh** [comment]
@reginapizza @iam-veeramalla @varshab1210 Please review

File: modules/gitops-release-notes-1-3-2.adoc [hunk 1]
```diff
@@ -0,0 +1,26 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+[id="gitops-release-notes-1-3-2_{context}"]
+= Release notes for {gitops-title} 1.3.2
+
+{gitops-title} 1.3.1 is now available on {product-title} 4.6, 4.7, 4.8, and 4.9.
```
Comment (iam-veeramalla) [inline modules/gitops-release-notes-1-3-2.adoc:8]: Please change the release to 1.3.2

File: modules/go-compatibility-and-support-matrix.adoc [hunk 2]
```diff
@@ -15,7 +15,7 @@ In the table, features are marked with the following statuses:
 |*OpenShift GitOps* 6+|*Component Versions*|*OpenShift Versions*|*Support status*
 
 |*Version*|*kam*|*Helm*|*Kustomize*|*Argo CD*|*ApplicationSet*|*Dex*||
-|1.3.0|0.0.40|3.6.0|4.2.0|2.1.2|0.2.0|2.28.0|4.6-4.9|GA
+|1.3.0|0.0.40|3.6.0|4.2.0|2.1.8|0.2.0|2.30.0|4.6-4.9|GA
```
Comment (iam-veeramalla) [inline modules/go-compatibility-and-support-matrix.adoc:18]: Dont we update patch releases here ?

1.3.0 does not support Dex 2.30.0, it is actually supported by 1.3.2. To be more clear I think we should update the dependency matrix for z-stream releases as well.

File: modules/gitops-release-notes-1-3-2.adoc [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+[id="fixed-issues-1-3-2_{context}"]
+== Fixed issues
+
+* Previously, in the `OperaturHub` if you filter by "Disconnected" under the *Infrastructure Features* the {gitops-title} did not appear, since it the Operator did not had the related annotation set in its CSV. The {gitops-title} is now annotated with "Disconnected Cluster" as an *Infrastructure Feature*.
```
Comment (iam-veeramalla) [inline modules/gitops-release-notes-1-3-2.adoc:13]: `since it` should be changed to `since`.

File: modules/gitops-release-notes-1-3-2.adoc [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+* When using an "Application-scoped" argocd (e.g. one that is not scoped to All Namepsaces in a cluster), the {gitops-title} dynamically maintains a list of managed namespaces. These namespaces includes the `argocd.argoproj.io/managed-by` label. This list of namespaces gets cached in argocd *Settings > Clusters > "in-cluster" > NAMESPACES*. However, if one of these namespaces gets deleted, the operator doesn't seem to notice, and the namespace remains in that list. This breaks both the *CONNECTION STATE* in that cluster config, and also all sync attempts after the namespace gets deleted. The sync will show errors. For example:
```
Comment (iam-veeramalla) [inline modules/gitops-release-notes-1-3-2.adoc:15]: I am now confused, do we call it `application-scoped` Argo CD or `namespace scoped` Argo CD. Can you tell me where did you find the phrase `application-scoped` Argo CD ?

File: modules/gitops-release-notes-1-3-2.adoc [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
++
+[source,text]
+----
+Argo service account does not have <random verb> on <random resource type> in namespace the-namespace-you-deleted. 
+----
++
+This bug is fixed.
```
Comment (iam-veeramalla) [inline modules/gitops-release-notes-1-3-2.adoc:22]: Please add JIRA IDs for all the bugs.

**Comment by iam-veeramalla** [comment]
@DebarghoGhosh I understand that we have upgraded the support matrix to reflect the Dex version upgrade as well as Argo CD version upgrade but can we also add this as a pointer to make it much visible.

**Comment by iam-veeramalla** [comment]
LGTM

**Review by iam-veeramalla — APPROVED** [review]

File: modules/gitops-release-notes-1-3-2.adoc [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+* Upgraded Dex to version *v2.30.0*
+
+[id="fixed-issues-1-3-2_{context}"]
+== Fixed issues
+
+The following issues were resolved in the current release:
+
+* Previously, in the `OperaturHub` if you filter by "Disconnected" under the *Infrastructure Features* the {gitops-title} did not appear, since the Operator did not had the related annotation set in its CSV. The {gitops-title} is now annotated with "Disconnected Cluster" as an *Infrastructure Feature*. link:https://issues.redhat.com/browse/GITOPS-1539[GITOPS-1539]
+
+* When using an "Namespace-scoped" argocd (e.g. one that is not scoped to All Namepsaces in a cluster), the {gitops-title} dynamically maintains a list of managed namespaces. These namespaces includes the `argocd.argoproj.io/managed-by` label. This list of namespaces gets cached in argocd *Settings > Clusters > "in-cluster" > NAMESPACES*. However, if one of these namespaces gets deleted, the operator doesn't seem to notice, and the namespace remains in that list. This breaks both the *CONNECTION STATE* in that cluster config, and also all sync attempts after the namespace gets deleted. The sync will show errors. For example:
```
Comment (missmesss) [inline modules/gitops-release-notes-1-3-2.adoc:26]:
```suggestion
* When using an "Namespace-scoped" argocd, for example, argocd that is not scoped to All Namepsaces in a cluster, the {gitops-title} dynamically maintains a list of managed namespaces. These namespaces include the `argocd.argoproj.io/managed-by` label. This list of namespaces is stored in a cache in argocd *Settings > Clusters > "in-cluster" > NAMESPACES*. Before this update, if you deleted one of these namespaces, the Operator ignored that, and the namespace remained in the list. This behavior broke the *CONNECTION STATE* in that cluster configuration, and all sync attempts resulted in errors. For example:
```
Most suggestions are based on the IBM SG. We should avoid using Latin abbreviations (e.g.) and using "cache" as a verb. Gets deleted is a bit colloquial. And we always describe previous problems with the past tenses.

File: modules/gitops-release-notes-1-3-2.adoc [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+* Upgraded Argo CD to version *2.1.8*
+
+* Upgraded Dex to version *2.30.0*
+
+[id="fixed-issues-1-3-2_{context}"]
+== Fixed issues
+
+The following issues have been resolved in the current release:
+
+* Previously, the {gitops-title} did not appear in the `OperatorHub` when you filtered by "Disconnected" under the *Infrastructure Features*, since the Operator did not have the related annotation set in its CSV. With this update, the {gitops-title} is annotated with "Disconnected Cluster" as an *Infrastructure Feature* link:https://issues.redhat.com/browse/GITOPS-1539[GITOPS-1539]
```
Comment (Preeticp) [inline modules/gitops-release-notes-1-3-2.adoc:24]: the {gitops-title} Operator
`Disconnected` We do not use double quotes to indicate GUI features/user entered values
CSV file?

Comment (Preeticp) [reply modules/gitops-release-notes-1-3-2.adoc:24]:
```suggestion
* Previously, in the OperatorHub UI under the *Infrastructure Features* section, when you filtered by `Disconnected` the {gitops-title} Operator did not show in the search results, as the Operator did not have the related annotation set in its CSV file. With this update, the `Disconnected Cluster` annotation has been added to the {gitops-title} Operator as an infrastructure feature. link:https://issues.redhat.com/browse/GITOPS-1539[GITOPS-1539]
```

Comment (Preeticp) [reply modules/gitops-release-notes-1-3-2.adoc:24]: the `Disconnected Cluster` - should this be `disconnected` see the [doc link](https://docs.openshift.com/container-platform/4.8/operators/operator_sdk/osdk-generating-csvs.html#olm-enabling-operator-for-restricted-network_osdk-generating-csvs) provided in [the issue](https://issues.redhat.com/browse/GITOPS-1446) )

File: modules/gitops-release-notes-1-3-2.adoc [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+
+* When using an "Namespace-scoped" argocd, for example, argocd that is not scoped to *All Namepsaces* in a cluster, the {gitops-title} dynamically maintains a list of managed namespaces. These namespaces include the `argocd.argoproj.io/managed-by` label. This list of namespaces is stored in a cache in argocd *Settings > Clusters > "in-cluster" > NAMESPACES*. Before this update, if you deleted one of these namespaces, the Operator ignored that, and the namespace remained in the list. This behavior broke the *CONNECTION STATE* in that cluster configuration, and all sync attempts resulted in errors. For example:
```
Comment (Preeticp) [inline modules/gitops-release-notes-1-3-2.adoc:26]: - a `Namespace-scoped` Argo CD instance, 
- example, an Argo CD instance that is...
- argocd - Argo CD names of products need to be mentioned as is with the appropriate case, space etc. Please check all the instances in the PR.
- the {gitops-title}/{gitops-title} - An article is not required in this case.
- argocd *Settings > Clusters > "in-cluster" > NAMESPACES*./*Argo CD -> Settings -> Clusters -> "in-cluster" -> NAMESPACES*. Please go through the OCP doc contributing guidelines: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference

File: modules/gitops-release-notes-1-3-2.adoc [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
++
+[source,text]
+----
+Argo service account does not have <random verb> on <random resource type> in namespace the-namespace-you-deleted.
```
Comment (Preeticp) [inline modules/gitops-release-notes-1-3-2.adoc:30]: Check System or software variable to be replaced by the user: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference Also see Supplementary style guide for more details.
<random_verb>
<random_resource_type>
the-namespace-you-deleted - add angular bracket and replace - with _

File: modules/gitops-release-notes-1-3-2.adoc [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+* When using an "Namespace-scoped" argocd, for example, argocd that is not scoped to *All Namepsaces* in a cluster, the {gitops-title} dynamically maintains a list of managed namespaces. These namespaces include the `argocd.argoproj.io/managed-by` label. This list of namespaces is stored in a cache in argocd *Settings > Clusters > "in-cluster" > NAMESPACES*. Before this update, if you deleted one of these namespaces, the Operator ignored that, and the namespace remained in the list. This behavior broke the *CONNECTION STATE* in that cluster configuration, and all sync attempts resulted in errors. For example:
++
+[source,text]
+----
+Argo service account does not have <random verb> on <random resource type> in namespace the-namespace-you-deleted. 
+----
++
+This bug is fixed. link:https://issues.redhat.com/browse/GITOPS-1521[GITOPS-1521]
+
+* {gitops-title} is annotated as "Deep Insights" under the *Capability* Level. link:https://issues.redhat.com/browse/GITOPS-1519[GITOPS-1519]
```
Comment (Preeticp) [inline modules/gitops-release-notes-1-3-2.adoc:35]: This should be The {gitops-title} Operator
*Deep Insights*
Suggest: With this update, the {gitops-title} Operator has been annotated with the *Deep Insights* capability level.

File: modules/gitops-release-notes-1-3-2.adoc [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+
+* Previously, the ArgoCD operator managed the `resource.exclusion` by itself but ignored the `resource.inclusion`. This prevented the Resource Inclusion configured in Argo CD CR to generate in the `argocd-cm configmap`. This bug is fixed. link:https://issues.redhat.com/browse/GITOPS-1518[GITOPS-1518]
```
Comment (Preeticp) [inline modules/gitops-release-notes-1-3-2.adoc:37]: Argo CD Operator
the `resource.exclusion`/`resource.inclusion` field
the `resource.inclusion` field configured in the `Argo CD` CR
`argocd-cm` configuration map

**Review by Preeticp — COMMENTED** [review]
@DebarghoGhosh PTAL at the suggestions.

**Comment by Preeticp** [comment]
Can you please confirm if this needs to be part of 4.8 as well? 4.9 and 4.10 seem to be the correct versions.

**Pull request merged**